### ISSUE TITLE
[Feature Request]: Disabling the "Saved to liked music." toast.

### ISSUE THREAD

**Issue by chocolime** [th-ch/youtube-music#2711]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

The mentioned toast persists indefinitely after a song has been added to your liked list, which is annoying. 

### Proposed Solution

I would like to ask for a toggle to enable/disable it.

### Alternatives Considered

Alternatively, it fading after a few seconds would work too. I think this should be the default in general but idk how difficult it'd be.

**Comment by Joey451-OG** [comment]
I have also noticed that the toast remains visible unless it is explicitly dismissed. Is this the intended behavior?

**Comment by chocolime** [comment]
> I have also noticed that the toast remains visible unless it is explicitly dismissed. Is this the intended behavior?

I think it so even on regular youtube music. :/

**Comment by michaeltcoyle** [comment]
i would love this, that notification annoys me way more than it should

**Comment by zdv80** [comment]
Yes Please! It would be great if it automatically disappears after a few seconds :D

**Comment by MilanMamangkey1** [comment]
For what it's worth, you can dismiss it by pressing the escape key. I wish I didn't have to do this and just have it be gone.

**Comment by JLSBeth** [comment]
I also wish this was implemented please

**Comment by kvn1351** [comment]
I've made a plugin and a subsequent PR for this: https://github.com/pear-devs/pear-desktop/pull/4642

Edit: didn't see that you had already made it :/
I would argue that my implementation is safer however, as it uses an observer instead and is correctly clears up timers.
